packages/sui-bundler/webpack.config.prod.js

```diff
@@ -108,50 +98,8 @@ module.exports = {
     new ManifestPlugin({
       fileName: 'asset-manifest.json'
     }),
-    when(
-      config.offline && (config.offline.fallback || config.offline.runtime),
-      () =>
-        new GenerateSW({
```

Comment:
I think there should be references to this in the README you are not removing

Reply:
Removed!